libraries/chain/database.cpp

```diff
@@ -1888,6 +1888,19 @@ namespace golos { namespace chain {
                 median_props.*param = active[active.size() / 2]->props.*param;
             };
 
+            auto calc_median_battery = [&](auto&& window, auto&& items) {
+                std::nth_element(
+                    active.begin(), active.begin() + active.size() / 2, active.end(),
+                    [&](const auto* a, const auto* b) {
+                        return a->props.*window / a->props.*items <
```

Comment:
@afalaleev Which of these 2 settings should be selected? What behaviour do you want in result?

Reply:
https://github.com/GolosChain/golos/issues/533#issuecomment-414747109

Reply:
@afalaleev Here is `window / items` (let's write "/" only with spaces - otherwise it could be misleaded with OR), not `std::tie`.
How could I use tie if it gives bad result here: https://pastebin.com/VwB470nz